packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -370,20 +373,52 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 		);
 	}
 
+	function addFeedbackAssistantMessage() {
+		const id = getRandomId();
+		chatMessages.value.push({
```

Comment:
why is not the backend returning this? The whole logic should live in the assistant service.

Reply:
With the latest feedback, we don't want to contact the AI unless users ask for another suggestion. We can update the service to just respond with the hard-coded message but this saves us from pinging the service at all.
Let me know if you strongly think this should be moved to service

Reply:
Yes this should live in the service. The whole chat flow should live there. Otherwise becomes really hard to change things because one version of n8n will always inject a message at this point.

Reply:
to clarify, we should bring back `node-execution-errored` event and update the response there